src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.scss

```diff
@@ -13,13 +13,32 @@
 
     color: var(--text-color);
 
-    font-weight: normal;
-    text-shadow: var(--gradient-black-rgba-with-08-opacity-color) 0 0 8px;
+    font-weight: bold;
+    text-shadow: var(--gradient-black-rgba-with-08-opacity-color) 1px 1px 3px;
     text-overflow: ellipsis;
     white-space: nowrap;
     z-index: 11;
   }
 
+  &.media-unavailable {
+    &:before {
+      content: '';
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      background: repeating-linear-gradient(
+                      45deg,
+                      #00000000,
+                      #00000000 12px,
+                      #9a0d0daa 12px,
+                      #9a0d0daa 18px
```

Comment:
Wild colors appears. We don't do that no more ;)

Reply:
They are no more! Vars introduced!